desktop-sync/latest/index.md

```diff
@@ -4,7 +4,7 @@ title: Alfresco Desktop Sync
 
 With Desktop Sync, desktop users can securely and automatically sync content between their desktop and Alfresco Content Services.
 
-This release of Desktop Sync is a compatibility release to support Alfresco Sync Service 3.7, and allows you to view the sync progress of your files and folders for a better user experience. See the **Sync** progress screen in [Windows]({% link desktop-sync/latest/using/select-to-sync.md %}#sync/windows/init-sync-win) and [Mac]({% link desktop-sync/latest/using/select-to-sync.md %}#sync/mac/init-sync-mac) for more details.
+This release of Desktop Sync is a compatibility release to support Alfresco Sync Service 3.7, and provides accessibility improvements in the Desktop Sync clients.
```

Comment:
@anxumalo I don't think we should mention accessibility in this release because we have made the changes only for MAC. For windows work is still pending. So, it will be sort of incomplete accessibility release. @lilustig can comment on this.

Reply:
Thanks @abhishekkumar31 - I've now removed it